src/modules.py

```diff
@@ -591,18 +590,11 @@ def parse_gluedropper(config: Dict[str, Any]) -> IocCollection:
 @module("neutrino")
 def parse_neutrino(config: Dict[str, Any]) -> IocCollection:
     iocs = IocCollection()
-    if "urls" in config:
-        for url in config["urls"]:
+    for url in config.get("urls", []):
             iocs.try_add_url(url)
```

Comment:
indent with 4 spaces (we don't have CI configured on github, but check with flake8 & mypy locally please)